### PR TITLE
dash/datagrid-improvements

### PR DESCRIPTION
**Pull request by karolkolodziej**
Made `datagrid` part of the dashboards.

### CODE REVIEW

File: ts/Dashboards/Plugins/DataGridTypes.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@
  *
  * */
 
-import type * as D from '@highcharts/datagrid';
+import type * as D from 'datagrid';
```
Comment (bre1470) [inline ts/Dashboards/Plugins/DataGridTypes.ts:25]: This has to be changed accordingly to the NPM distribution.

File: test/typescript-dts/dashboards/tsconfig.json [hunk 2]
```diff
@@ -16,8 +16,8 @@
             "@highcharts/dashboards/*": ["code/dashboards/*"],
             "highcharts": ["code/highcharts.src"],
             "highcharts/*": ["code/*"],
-            "@highcharts/datagrid": ["code/datagrid/datagrid.src"],
-            "@highcharts/datagrid/*": ["code/datagrid/*"],
+            "datagrid": ["code/datagrid/datagrid.src"],
+            "datagrid/*": ["code/datagrid/*"],
```
Comment (bre1470) [inline test/typescript-dts/dashboards/tsconfig.json:20]: Same here.

**Review by bre1470 — CHANGES_REQUESTED** [review]
If there will be no NPM package for DataGrid, every reference to it can be removed and the existing `@highcharts/dashboards` reference used instead.

**Review by bre1470 — APPROVED** [review]
Thanks! LGTM 👍

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**